packages/graphics/mesa/package.mk

```diff
@@ -13,6 +13,14 @@ PKG_LONGDESC="Mesa is a 3-D graphics library with an API."
 PKG_TOOLCHAIN="meson"
 PKG_BUILD_FLAGS="+lto"
 
+case $PROJECT in
+  Amlogic)
```

Comment:
I recommend this to be changed to check if `GRAPHIC_DRIVERS` contains `lima` or `panfrost`, both Allwinner and Rockchip will probably want to use same version.